### DISCUSSION TITLE
Customize Select component with images and text

### DISCUSSION THREAD

**Discussion by DomenicoFormoso** [Megabit/Blazorise#5538] [General]
Can you customize Select component and draw an image into the element, like for Transfer List component using RenderFragment?
If that's not possible, might it be a good idea for a new feature?

**Comment by stsrki** [comment]
We use native `select` html elements, so it is not that easy to customize. But, you can try. Here is an example that might help you get started https://stackoverflow.com/questions/2965971/how-to-add-images-in-select-list
